services/discovery/Makefile

```diff
@@ -17,14 +17,17 @@ COVERAGE_REPORT = coverage-report
 FLAKE8_OUT = flake8.out
 BUILD_DIR = .aws-sam
 DEPLOYMENT_FILE = .deploy
+ACCOUNT_ID = $(shell source ../../project.sh && cz_assert_profile &> /dev/null && \
+							 aws sts get-caller-identity | jq -r -e '.Account')
```

Comment:
Ah, glad you are using profiles this way.  In older makefiles we used to have an annoying problem where AWS operations would randomly fail because we were not asserting a session.